### ISSUE TITLE
Revit Analysis Display

### ISSUE THREAD

**Issue by fraguada** [mcneel/rhino.inside-revit#57]
We've had a request to support passing analysis data back into Revit from tools like Ladybug. Can we take a look at supporting Analysis Display?

- https://apidocs.co/apps/revit/2020/958e2e12-587d-f188-5d7b-f13d7dbfdf48.htm
- https://knowledge.autodesk.com/support/revit-products/learn-explore/caas/CloudHelp/cloudhelp/2019/ENU/Revit-Analyze/files/GUID-84A57C91-14D4-406F-A9F6-972B21029521-htm.html

An example from Paul Wintour:
https://www.linkedin.com/posts/paul-wintour_for-years-now-aec-professionals-have-struggled-activity-6585392189738745856-kykD SEPP65 solar access using Rhino Inside https://vimeo.com/363977394

**Comment by gschleusner** [comment]
We did a version of this ... a long time ago.... manually,  I think we even make a format to get the data from Diva or Ladybug.  https://github.com/HOKGroup/HOK-Revit-Addins/tree/master/AVF%20Manager   I doubt there is much here that is useful from a code perspective.  Maybe a pattern or two.

**Comment by scotttd** [comment]
Does this help with this issue: https://discourse.mcneel.com/t/export-geometry-mesh-from-rhino-to-revit-through-rhino-inside-revit/86477/5

**Comment by fraguada** [comment]
@scotttd Not necessarily. @kike-garbo enabled the previewing of meshes with vertex colors in Revit, but the Analysis Display in Revit actually adds the data to the object, if I understand that correctly. So what we have currently is a useful addition, but it is probably not the best performing, not the most idiomatic manner to store this data with the Revit model (as I understand it).
